tools/dockerfile/test/bazel/Dockerfile

```diff
@@ -48,6 +49,27 @@ RUN pip install virtualenv
 RUN pip install futures==2.2.0 enum34==1.0.4 protobuf==3.5.2.post1 six==1.10.0 twisted==17.5.0
 
 
+#=================
+# Compile CPython 3.6.9 from source
```

Comment:
I'm not a fan of building stuff from source in our docker images (as it makes rebuilding them slow). Is there a better way?

Reply:
perhaps there's an updated version of gcr.io/oss-fuzz-base/base-builder base image we could use?

Reply:
I agree building is slow. It is mitigated by Docker caching the result image. This PR already using their latest base image, underlying it is Ubuntu 16.04 (see [their repo](https://github.com/google/oss-fuzz/tree/master/infra/base-images/base-builder)).

The Python 3 version distributed by `apt` in 16.04 is `3.5.1`. Better ideas?